lib/gringotts/gateways/sagepay.ex

```diff
@@ -0,0 +1,362 @@
+defmodule Gringotts.Gateways.Sagepay do
+  @moduledoc """
+  [sagepay][home] gateway implementation.
+
+  The following features of MONEI are implemented:
+
+  | Action                       | Method        | 
+  | ------                       | ------        | 
+  | Authorize                    | `authorize/3` | 
+  | Release                      | `capture/3`   | 
+  | Refund                       | `refund/3`    | 
+  | Reversal                     | `void/2`      | 
+  | Purchase                     | `purchase/3`  | 
+  | Tokenization / Registrations | `store/2`     | not done yet
+
+
+
+  ## Instructions!
```

Comment:
This was an instruction, read once then remove. We cannot merge it in this state can we?